handbook/engineering/README.md

```diff
@@ -546,6 +575,15 @@ Please see [handbook/engineering#accept-new-apple-developer-account-terms](https
 ##### Merging during the freeze period
 Please see [handbook/engineering#merge-a-pull-request-during-the-freeze-period](https://fleetdm.com/handbook/engineering#merge-a-pull-request-during-the-freeze-period)
 
+##### Merge a bug fix during the freeze period
```

Comment:
```suggestion
##### Merge a bug fix during the QA period
```

Reply:
Second thought QA period may be ambivalent. Need netter wording.

Reply:
These are stubs to cover the old header links so we want to keep them as the old header.